public/docs/_examples/template-syntax/dart/lib/little_tour.dart

```diff
@@ -0,0 +1,23 @@
+library template_syntax.little_tour;
+
+import 'package:angular2/angular2.dart';
+
+@Component(selector: 'little-tour')
+@View(
+    template: '''<h4>Little Tour of Heroes</h4>
+    <input #new-hero
+      (keyup.enter)="addHero(newHero)"
+      (blur)="addHero(newHero)">
+    <button (click)=addHero(newHero)>Add</button>
+    <ul><li *ng-for="#hero of heroes">{{hero}}</li></ul>''',
+    directives: const [CORE_DIRECTIVES, FORM_DIRECTIVES])
+class LittleTourComponent {
+  List<String> heroes = ['Windstorm', 'Bombasto', 'Magneta', 'Tornado'];
+
+  addHero(newHero) {
```

Comment:
Awesome. The return type should also be `void`. Are you going to update this pull request?